Network Class/Network.cpp

```diff
@@ -138,6 +138,39 @@ void Network::updateWeightsAndBiases()
 
 }
 
+//moves backwards through Network calculating error at each level and using that to increment sumNablaB and sumNablaW
+//requires the assumption that expected values has an int
+bool backProp(int index)
+{
+	int lastInd = numLayers - 1;
+	int lastSize = layerSizes[lastInd];
+	Vector expectedValues = getAt<int>(expectedValuesInfile, index);
+	
+	bool correct = true;
+	// to write: check if activations[lastInd] matches expectedVal at index and set correct accordingly
+
+	// Matrix costPrime = activations[lastInd] - expectedValues;
+	// above statement needs expectedValues to be a matrix to work
+	// consider the implications of having getAt return a (j x 1) matrix instead of a vector
```

Comment:
We can overload getAt(), so we can use it to assign the weightedInputs and the matrices if needed!